packages/space-opera/src/components/animation_controls/reducer.ts

```diff
@@ -15,34 +15,22 @@
  *
  */
 
-import {registerStateMutator, State} from '../../space_opera_base.js';
+import {Action, State} from '../../types.js';
 
-/** Set auto play enabled or not */
-const SET_AUTOPLAY_ENABLED = 'SET_AUTOPLAY_ENABLED';
-export const dispatchAutoplayEnabled = registerStateMutator(
-    SET_AUTOPLAY_ENABLED, (state: State, enabled?: boolean) => {
-      state.config = {...state.config, autoplay: !!enabled};
-    });
+const SET_ANIMATION_NAMES = 'SET_ANIMATION_NAMES';
+export function dispatchSetAnimationNames(animationNames: string[]) {
```

Comment:
Every action (for the most part) now looks like this, where they are returning an object with a type, and optional payload

Reply:
yeah, it is, it is the Action type, I can add them in a follow-up PR easy-enough